layouts/_default/baseof.html

```diff
@@ -47,13 +47,15 @@
     {{ end }}
 
     {{ block "header" . }}
+    {{ if (ne ($.Param "header_visible") false) }}
```

Comment:
I can't remember the difference between the two params syntaxes, but I'd prefer you use the same syntax that I'm using for the description a few lines below:

```
{{ if (ne .Site.Params.header_visible false) }}
```